fiftyone/plugins/skills.py

```
@@ -0,0 +1,159 @@
+        """The absolute path to the skill file."""
+        return self._path
+
+    @property
+    def plugin_name(self):
+        """The name of the plugin that provides this skill."""
+        return self._plugin_name
+
+    @property
+    def content(self):
+        """The full raw content of the skill file."""
+        with open(self._path, "r") as f:
```

Ruff (0.15.7)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)